vendor/bat-native-ads/src/bat/ads/internal/ads/serving/eligible_ads/exclusion_rules/conversion_exclusion_rule.cc

```diff
@@ -20,9 +20,8 @@ constexpr int kConversionCap = 1;
 
 bool DoesRespectCap(const AdEventList& ad_events,
                     const CreativeAdInfo& creative_ad) {
-  const int count = std::count_if(
-      ad_events.cbegin(), ad_events.cend(),
-      [&creative_ad](const AdEventInfo& ad_event) {
+  const size_t count = base::ranges::count_if(
+      ad_events, [&creative_ad](const AdEventInfo& ad_event) {
         return ad_event.confirmation_type == ConfirmationType::kConversion &&
                ad_event.creative_set_id == creative_ad.creative_set_id;
       });
```

Comment:
we can convert 
`constexpr int kConversionCap = 1;` to
`constexpr size_t kConversionCap = 1;`

Reply:
I have another follow up PR to fix this throughout brave ads

Reply:
Changed to int as will fix types in another PR